### ISSUE TITLE
Error occurred when setting both console logger and file logger

### ISSUE THREAD

**Issue by cq674350529** [qilingframework/qiling#701]
***Describe the bug**
Currently, when setting the console logger and file logger, the following error is generated.
```python
~/qiling/qiling/examples$ python3 hello_mips32el_linux_debug.py 
[+]	Profile: Default
--- Logging error ---
Traceback (most recent call last):
  File "/usr/lib/python3.6/logging/__init__.py", line 994, in emit
    msg = self.format(record)
  File "/usr/lib/python3.6/logging/__init__.py", line 840, in format
    return fmt.format(record)
  File "~/qiling/qiling/utils.py", line 86, in format
    record.levelname = self.get_level(record)
  File "~/qiling/qiling/utils.py", line 83, in get_level
    return LEVEL_NAME[record.levelname]
KeyError: '\x1b[95m[+]\x1b[0m'
Call stack:
  File "hello_mips32el_linux_debug.py", line 12, in <module>
    ql = Qiling(["rootfs/mips32el_linux/bin/mips32el_hello_static"], "rootfs/mips32el_linux", output="debug", log_file="./test.log")
  File "~/qiling/qiling/core.py", line 184, in __init__
    self.log.debug(debugmsg)
Message: 'Profile: Default'
Arguments: ()
[+]	load 0x400000 - 0x48d000

// ...
```

It seems that the console logger(`logging.StreamHandler()`) and file logger(`logging.FileHandler()`) share the same `LogRecord`, and the `record.levelname` will be changed in console logger first, then `LEVEL_NAME[record.levelname]` in file logger will generate an KeyError.

**Sample Code**
Just add an option `log_file="./test.log"` in `examples/hello_mips32el_linux_debug.py` to reproduce this issue.
```python
import sys

sys.path.append("..")
from qiling import *

if __name__ == "__main__":
    ql = Qiling(["rootfs/mips32el_linux/bin/mips32el_hello_static"], "rootfs/mips32el_linux", output="debug", log_file="./test.log")
    ql.run()
```

**Comment by xwings** [comment]
@wtdcode all yours

**Comment by wtdcode** [comment]
Fixed in 2fc8ed83
